diff --git a/lib/util/mayContainChildComponent.js b/lib/util/mayContainChildComponent.js
index 9c12846d93a699b3870fcc682b70318d549a9b11..4caeaf164359eef53cad3e3f167fed263e7fa40a 100644
--- a/lib/util/mayContainChildComponent.js
+++ b/lib/util/mayContainChildComponent.js
@@ -5,7 +5,7 @@ Object.defineProperty(exports, "__esModule", {
 });
 exports["default"] = mayContainChildComponent;
 var _jsxAstUtils = require("jsx-ast-utils");
-var _minimatch = _interopRequireDefault(require("minimatch"));
+var _minimatch = { default: require("minimatch").minimatch };
 function _interopRequireDefault(e) { return e && e.__esModule ? e : { "default": e }; }
 /**
  * Returns true if it can positively determine that the element lacks an
diff --git a/lib/util/mayHaveAccessibleLabel.js b/lib/util/mayHaveAccessibleLabel.js
index 07eb76610957454796da190d118f1b1061bbad20..2f7a70fc99d1d172b6b8252b51cea5110909e838 100644
--- a/lib/util/mayHaveAccessibleLabel.js
+++ b/lib/util/mayHaveAccessibleLabel.js
@@ -6,7 +6,7 @@ Object.defineProperty(exports, "__esModule", {
 exports["default"] = mayHaveAccessibleLabel;
 var _arrayIncludes = _interopRequireDefault(require("array-includes"));
 var _jsxAstUtils = require("jsx-ast-utils");
-var _minimatch = _interopRequireDefault(require("minimatch"));
+var _minimatch = { default: require("minimatch").minimatch };
 function _interopRequireDefault(e) { return e && e.__esModule ? e : { "default": e }; }
 /**
  * Returns true if a labelling element is found or if it cannot determine if
